src/Provider/SymfonyUsageProvider.php

```diff
@@ -216,6 +229,44 @@ private function getMethodUsagesFromReflection(InClassNode $node): array
         return $usages;
     }
 
+    /**
+     * @return list<ClassMethodUsage>
+     */
+    private function getMethodUsagesFromAttributeReflection(InClassMethodNode $node, Scope $scope): array
+    {
+        $usages = [];
+
+        foreach ($node->getMethodReflection()->getParameters() as $parameter) {
+            foreach ($parameter->getAttributes() as $attributeReflection) {
+                if ($attributeReflection->getName() === 'Symfony\Component\DependencyInjection\Attribute\AutowireLocator' || $attributeReflection->getName() === 'Symfony\Component\DependencyInjection\Attribute\AutowireIterator') {
+                    $arguments = $attributeReflection->getArgumentTypes();
+
+                    if (!isset($arguments['services']) || !isset($arguments['defaultIndexMethod'])) {
+                        continue;
+                    }
+
+                    $className = $arguments['services']->getConstantStrings();
```

Comment:
I believe `services` can be array or strings and `getConstantStrings` would return `[]` in such case. Test needed.